src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -262,12 +263,12 @@ const useGroupInfo = (channelId = null) => {
   };
 
   const onAddMember = async (selectedUsers) => {
-    const responseChannelData = await addMemberGroup({
-      channelId,
-      memberIds: selectedUsers.map((user) => user.user_id)
-    });
-
     try {
+      setIsLoadingAddMember(true);
+      const responseChannelData = await addMemberGroup({
+        channelId,
+        memberIds: selectedUsers.map((user) => user.user_id)
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are appropriate as they wrap the `addMemberGroup` function call within a try-catch block and set the `isLoadingAddMember` state before and after the operation. This ensures that any errors during the operation are caught and handled appropriately, and the loading state is correctly managed.
```diff
+ try {
+   setIsLoadingAddMember(true);
+   const responseChannelData = await addMemberGroup({
+     channelId,
+     memberIds: selectedUsers.map((user) => user.user_id)
+   });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->